.github/workflows/upd-client-launcher.yml

```diff
@@ -0,0 +1,115 @@
+# To be able use this workflow you must ensure that your repository have enable write and right access for GitHub Actions.
+#
+# GUI Reference to enable access:
+# Repository --> Actions --> General --> Workflow permissions --> Read and write permissions
+#                                                            \--> Allow Github Actions to create and approve pull requests
+
+name: Update XNA CnCNet Client Launcher
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: "0 0 * * *" # Starts everyday at 00:00 UTC+0
+
+env:
+  CLIENT_PATH: package # Do not add trailing slash
+  LAUNCHER_NAME: CnCNetYRLauncher.exe # Full launcher filename
+  ICON_NAME: clienticon.ico # Full icon filename in the Resources folder
+  UPDATE_BRANCH_NAME: update-client-launcher # The workflow adds the release tag after this value, i.e. "update-client-launcher-2.12.13"
+  TARGET_BRANCH: develop # Into this branch workflow should create pull request for merge
+  COMMIT_MESSAGE: Update client launcher to the latest version
+  PR_TITLE: Update client launcher to the version # The workflow adds the release tag after this value, i.e. "Update client binaries to the launcher 2.12.13"
+  PR_BODY: This is an automatic pull request to update client launcher to the latest release published in [CnCNet/xna-cncnet-client-launcher](https://github.com/CnCNet/xna-cncnet-client-launcher) repository.
+
+jobs:
+  publish-update-pr:
+    runs-on: windows-2022
+
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v3
+      with:
+        lfs: true
+    
+    - name: Download Latest Release
+      id: download
+      uses: robinraju/release-downloader@v1
+      with:
+        repository: CnCNet/xna-cncnet-client-launcher
+        latest: true
+        fileName: CnCNet*
+
+    - name: Download Icon Changer
```

Comment:
I suggest download a fixed version like [1.1.0](https://github.com/stefanGaina/icon-changer/releases/tag/v1.1.0) instead of latest. Manually bump the version number if the new version seems functional and not harmful